protocol/avdl/keybase1/metadata.avdl

```diff
@@ -55,7 +55,7 @@ protocol metadata {
   ChallengeInfo getChallenge();
   int authenticate(string signature);
   void putMetadata(MDBlock mdBlock, KeyBundle readerKeyBundle, KeyBundle writerKeyBundle, map<string> logTags, union{null, LockContext} lockContext, MDPriority priority);
-  MetadataResponse getMetadata(string folderID, bytes folderHandle, string branchID, boolean unmerged, long startRevision, long stopRevision, map<string> logTags, union{null, LockID} lockBeforeGet);
+  MetadataResponse getMetadata(string folderID, bytes folderHandle, string branchID, boolean unmerged, long startRevision, long stopRevision, map<string> logTags, union{null, LockID} lockBeforeGet, boolean noCreate);
```

Comment:
I'm fine with a bool, but just for the sake of discussion: any chance we'll add more behaviors later, and we should make this a more general options flag/struct?

Reply:
Good point; I'll change it into an enum.